server/lib/tuist/application.ex

```diff
@@ -80,6 +80,7 @@ defmodule Tuist.Application do
         Tuist.Vault,
         {Oban, Application.fetch_env!(:tuist, Oban)},
         {Cachex, [:tuist, []]},
+        Tuist.Cache.KeyValueStore,
```

Comment:
this should be moved to the new elixir project once we have it (and not started in our main application)